step/src/main/java/servlets/MatchingServlet.java

```diff
@@ -23,9 +29,25 @@
 @WebServlet("/matching")
 public class MatchingServlet extends HttpServlet {
 
+  public static String REQUEST_TYPE = "requestType";
+  public static String REQUEST_TYPE_MATCH = "match-request";
```

Comment:
nit: it might be better style to use either camel-case of dash-separated words for both strings

Reply:
Yeah I was debating myself about this. I saw REQUEST_TYPE more as a variable name (so camel case) but I'll go ahead and make it consistent